documentation/src/main/asciidoc/topics/con_server_hinting.adoc

```diff
@@ -1,25 +1,23 @@
 [id='server-hinting_{context}']
 = Server hinting
 
-The following topology hints can be specified:
-
-Machine::
-  This is probably the most useful, when multiple JVM instances run on the
-  same node, or even when multiple virtual machines run on the same physical machine.
-Rack::
-  In larger clusters, nodes located on the same rack are more likely to experience a
-  hardware or network failure at the same time.
-Site::
-  Some clusters may have nodes in multiple physical locations for extra resilience.
-  Note that Cross site replication is another alternative for
-  clusters that need to span two or more data centres.
-
-All of the above are optional.
-When provided, the distribution algorithm will try to spread the ownership of each segment across as many sites, racks, and machines (in this order) as possible.
+Server hinting increases availability of data in distributed caches by replicating entries across as many servers, racks, and data centers as possible.
+
+NOTE: Server hinting applies only to distributed caches because it distributes data across the physical topology of your {brandname} cluster.
+
+When {brandname} distributes the copies of your data, it follows the order of precedence: site, rack, machine, and node.
+All of the configuration attributes are optional.
+For example, when you specify only the rack IDs, then {brandname} distributes the copies across different racks and nodes.
+
+Server hinting can impact cluster rebalancing operations by moving more segments than necessary if the number of segments for the cache is too low.
+
+TIP: An alternative for clusters in multiple data centers is cross-site replication.
 
 [discrete]
 == Server hinting configuration
 
+To configure server hinting for a distributed cache, you provide information about the physical topology as part of the transport configuration for the Cache Manager:
```

Comment:
I'd probably just drop this sentence. The "To configure.." construction should lead into a procedure and signal to users that they are about to perform a task.

I also think it's obvs from the configuration examples that the info about the physical topology is included in the transport config. IMO there's no need to explicitly state that.